error-prone-contrib/pom.xml

```
@@ -47,6 +47,10 @@
             <artifactId>jackson-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
```

We should omit this one from the runtime classpath:
```suggestion
            <artifactId>jackson-core</artifactId>
            <scope>provided</scope>
```